arcade/draw_commands.py

```diff
@@ -706,10 +741,15 @@ def draw_rectangle_outline(center_x: float, center_y: float, width: float,
     _generic_draw_line_strip(point_list, color, gl.GL_TRIANGLE_STRIP)
 
 
+@warning(
+    message=\
+        "Drawing with lrtb(left, right, top, bottom) is deprecated use lrbt instead i.e draw_lrbt_rectangle_filled.",
+    warning_type =NameChangeWarning,
+)
```

Comment:
Same applies here. Somewhat shorter and more explicit message is probably good.

Reply:
Updated in https://github.com/pythonarcade/arcade/commit/f421cb9a1ed17d82a3c817eaac4fb60fca2a223e